File: platforms/flutter/lib/src/category.dart
class Category {
  final String slug;
  final String displayName;
  final int count;

  const Category({
    required this.slug,
    required this.displayName,
    required this.count,
  });

  @override
  bool operator ==(Object other) =>
      other is Category && other.slug == slug;

  @override
  int get hashCode => slug.hashCode;

  @override
  String toString() => 'Category($slug, count: $count)';
}

File: Sources/IdnFinLogos/Category.swift
import Foundation

public struct Category: Sendable, Equatable, Hashable {
    public let slug: String
    public let displayName: String
    public let count: Int

    public init(slug: String, displayName: String, count: Int) {
        self.slug = slug
        self.displayName = displayName
        self.count = count
    }
}
